docs/source/privacy/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Data Collection"
+sidebar_title: "Data Collection"
```

Comment:
You can leave this out if it matches the `title` exactly. Only required to override what shows up on the left